flexget/plugins/sites/lostfilm.py

```diff
@@ -47,147 +37,128 @@ class LostFilm(object):
 
       lostfilm:
         url: <url>
+        pages: <number>
     """
 
     schema = {
         'type': ['boolean', 'object'],
         'properties': {
             'url': {'type': 'string', 'format': 'url'},
+            'pages': {'type': 'integer'}
         },
         'additionalProperties': False
     }
 
     def build_config(self, config):
-        """Set default url to config"""
+        """Set default url and pages config"""
         if isinstance(config, bool) and config is True:
-            config = {'url': LOSTFILM_URL}
+            config = {'url': NEW_RELEASES_URL, 'pages': 3}
         return config
 
     def on_task_input(self, task, config):
         config = self.build_config(config)
         if config is False:
             return
-        try:
-            rss = feedparser.parse(config['url'])
-        except:
-            raise PluginError('Cannot parse rss feed')
-        if rss.get('status') != 200:
-            raise PluginError('Received not 200 (OK) status')
-        entries = []
-        for item in rss.entries:
-            if item.get('link') is None:
-                log.debug('Item doesn\'t have a link')
-                continue
-            if item.get('description') is None:
-                log.debug('Item doesn\'t have a description')
-                continue
-            try:
-                lostfilm_num = LOSTFILM_ID_REGEXP.search(
-                    item['description']).groups()
-            except:
-                log.debug('Item doesn\'t have lostfilm id in description')
-                continue
-            try:
-                season_num, episode_num = [
-                    int(x) for x in EPISODE_REGEXP.search(
-                        item['link']).groups()]
-            except:
-                log.debug('Item doesn\'t have episode id in link')
-                continue
-            params = {
-                'c': lostfilm_num,
-                's': season_num,
-                'e': episode_num
-            }
-            redirect_url = 'http://www.lostfilm.tv/v_search.php'
-            try:
-                response = task.requests.get(redirect_url, params=params)
-            except RequestException as e:
-                log.error('Could not connect to redirect url: {:s}'.format(e))
-                continue
-
-            page = get_soup(response.content)
-            try:
-                redirect_url = page.head.meta['content'].split('url=')[1]
-            except:
-                log.error('Missing redirect')
-                continue
 
+        entries = []
+        for page_num in range(1, config['pages']):
+            # Fetch release page
+            release_url = '{:s}/page_{:d}'.format(config['url'], page_num)
+            log.debug('Fetching page {:d} of {:d}'.format(page_num, config['pages']))
```

Comment:
Let the logger handle string interpolation ie. change it to `log.debug('Fetching page %s of %s', page_num, config['pages'])`